<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Countries</class>
 <widget class="QDialog" name="Countries">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>549</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>549</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>421</width>
    <height>549</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leServers">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Your Server IP: 127.0.0.1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChoose">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>460</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose</string>
   </property>
  </widget>
  <widget class="QLabel" name="lightTheme">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>421</width>
     <height>561</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resource.qrc">:/img/whiteTheme.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="darkTheme">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>551</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resource.qrc">:/img/11zon_cropped.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leServersName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Your Server Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbFieldsCheck">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #f47c00;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; color:#f47c00;&quot;&gt;Fill The Fields&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="lwServers">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>361</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <zorder>darkTheme</zorder>
  <zorder>lightTheme</zorder>
  <zorder>btnBack</zorder>
  <zorder>leServers</zorder>
  <zorder>btnAdd</zorder>
  <zorder>btnDelete</zorder>
  <zorder>btnChoose</zorder>
  <zorder>leServersName</zorder>
  <zorder>lbFieldsCheck</zorder>
  <zorder>lwServers</zorder>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
